mods_cocina_mappings/mods_to_cocina_relatedItem.txt

```diff
@@ -0,0 +1,136 @@
+1. Related item with type
+<relatedItem type="series">
+  <titleInfo>
+    <title>Lymond chronicles</title>
+  </titleInfo>
+  <name type="personal">
+    <namePart>Dunnett, Dorothy</namePart>
+  </name>
+  <physicalDescription>
+    <extent>6 vols.
+  </physicalDescription>
+</relatedItem>
+{
+  "relatedResource": [
+    {
+      "title": [
+        {
+          "value": "Lymond chronicles"
+        }
+      ],
+      "contributor": [
+        {
+          "type": "person",
+          "name": [
+            {
+              "value": "Dunnett, Dorothy"
+            }
+          ]
+        }
+      ],
+      "form": [
+        {
+          "value": "6 vols.",
+          "type": "extent"
+        }
+      ],
+      "type": "series"
+    }
+
+  ]
+}
+
+
+2. Related item without type
+<relatedItem>
+  <titleInfo>
+    <title>Supplement</title>
+  </titleInfo>
+  <abstract>Additional data.</abstract>
+</relatedItem>
+{
+  "relatedResource": [
+    {
+      "title": [
+        {
+          "value": "Supplement"
+        }
+      ]
+    }
+  ],
+  "note": [
+    {
+      "value": "Additional data.",
+      "type": "summary"
+    }
+  ]
+}
+
+3. Related item without title
+<relatedItem>
+  <location>
+    <url>https://www.example.com</url>
+  </location>
+</relatedItem>
+{
+  "relatedResource": [
+    {
+      "access": {
+        "url": [
+          {
```

Comment:
Why are url's entities rather than just strings?

Reply:
Because they might be qualified in some way - for example, marking one as primary or linking a URL that is for a specific part of a described resource. Or different URLs might have different display labels. See the Rumsey and web archive examples in the location mapping for use cases.